### PR TITLE
Prevent large fixtures download from failing

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?154131
The GET request overflowed, so I switched it to a POST request, and also removed unnecessary request params from being sent over.

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
I was surprised not to see any view changes with this - how is that getting handled?

**Comment by esoergel** [comment]
Whoops, you're right, I was thinking because it's sent over in the body of the request it would go to the same request.REQUEST pseudo dict, but it was actually just failing silently.  Fixing now.

**Comment by esoergel** [comment]
okay, should be g2g now

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/fixtures/views.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ def download_item_lists(request, domain):
     download = DownloadBase()
     download.set_task(fixture_download_async.delay(
         prepare_fixture_download,
-        table_ids=request.GET.getlist("table_id"),
+        table_ids=request.POST.getlist("table_ids[]", []),
```
Comment (czue) [inline corehq/apps/fixtures/views.py]: i hadn't seen this before so looked it up: http://stackoverflow.com/a/11190754/8207

yuck

Comment (esoergel) [inline corehq/apps/fixtures/views.py]: yeah, pretty gross.  You can disable it by passing `traditional=true` or something to `$.ajax`, but if this is the convention nowadays...

**Pull request merged**